src/ossf_scorecard/scorecard_cli.py

```diff
@@ -0,0 +1,33 @@
+from google.cloud import bigquery
+
+client = bigquery.Client()
+
+query = """
+SELECT
+  date,
+  repo.name AS repo_name,
+  scorecard,
+  check.name AS check_name,
+  check.score AS check_score,
+  check.reason AS check_reason
+FROM
+  `openssf.scorecardcron.scorecard-v2`,
+  UNNEST(checks) AS check
+WHERE
+  repo.name = "github.com/pallets/flask"
+
+  LIMIT 100
+"""
+# AND scorecard.version = "v4.13.1-242-g153e06d9"
+
+query_job = client.query(query)
+
+results = query_job.result()
```

Comment:
Can this actually be tested? Do we need to mock api calls?

Reply:
have removed cli code for now, will use it when it is fully ready